limitiations: describe current limitations on autonegotations (#99)
bisdn/bisdn.github.io: limitations.md

- The 10G ports on [AS4610](https://www.edge-core.com/productsList.php?cls=1&cls2=9&cls3=46) only support advertising 1G, so the speed will be limited to 1G regardless of the link partner's ability.

> - There is an issue in the current Broadcom SDK where the 10G ports on [AS4610](https://www.edge-core.com/productsList.php?cls=1&cls2=9&cls3=46) will not transfer packets when autonegotiating down to 1G after being configured for 10G. Forcing the speed to 1G with disabled autonegotiation avoids this issue.

Reviewer: Terms like "current Broadcom SDK" should be avoided if possible. Even if the documentation were kept up-to-date at all times _and_ the reader knew that, it would be difficult for them to ascertain which SDK versions are affected, i.e. if their idea of the current version is the same. If either of these conditions is not met, the phrasing is even more confusing.

Author: How about something like
```suggestion
- Due to an upstream driver issue, the 10G ports on [AS4610](https://www.edge-core.com/productsList.php?cls=1&cls2=9&cls3=46) will not transfer packets when autonegotiating down to 1G after being configured for 10G. Forcing the speed to 1G with disabled autonegotiation avoids this issue.
```

Reviewer: The suggested change removes two pieces of information: a) that the issue is with the Broadcom SDK and b) that this is a recent regression. Sure, that's one way of keeping people from wondering which SDK versions are affected.

> - The 25G ports on [AG5648](https://agema.deltaww.com/product-info.php?id=41) do not support simultaneous detection of 1G with SGMII and 1G with KX, and will treatit as 1G with KX. If you use 1G SFP modules, configure the port to a fixed speed with 1G to work around this.

Reviewer 2: ```suggestion
- The 25G ports on [AG5648](https://agema.deltaww.com/product-info.php?id=41) do not support simultaneous detection of 1G with SGMII and 1G with KX, and will treat it as 1G with KX. If you use 1G SFP modules, configure the port to a fixed speed with 1G to work around this.
```

Reviewer 3: has been fixed
